Mobile/starter_project_matador/lib/features/example/presentation/widgets/signup_button.dart

```diff
@@ -0,0 +1,29 @@
+import "package:flutter/material.dart";
+
+class Button extends StatelessWidget {
```

Comment:
is there another better naming for this class? may be something like customButton or like that?

Reply:
done resolving it